contactsummary/js/src/contact-summary/controllers/key-details.controller.js

```diff
@@ -15,10 +15,15 @@ define([
     var vm = this;
 
     vm.ready = false;
+    vm.options = {};
 
     (function init () {
-      getContacts();
-      pubSub.subscribe('contract-refresh', resetKeyDetails);
+      initContractOptions()
+        .then(function () {
+          getContacts();
+        });
+
+      initSubscribers();
```

Comment:
Can we name this `initListeners`? same is used in L&A, it will be consistent then.

Reply:
sure we can!